backend/imap/conn.go

```diff
@@ -31,39 +43,34 @@ type conns struct {
 }
 
 func (b *conns) connect(username, password string) (email string, err error) {
-	var c *imap.Client
+	var c *imapclient.Client
 	if b.config.Tls {
-		c, err = imap.DialTLS(b.config.Host(), nil)
+		c, err = imapclient.DialTLS(b.config.Host(), nil)
 	} else {
-		c, err = imap.Dial(b.config.Host())
+		c, err = imapclient.Dial(b.config.Host())
 	}
 	if err != nil {
 		return
 	}
 
-	//c.SetLogMask(imap.LogAll)
-
 	if !b.config.Tls {
```

Comment:
Nope, it's used to distinguish between TLS and STARTTLS connections. Basically, with TLS connections all the traffic is encrypted from the start, but with STARTTLS the connection is unencrypted and a command is used to begin the TLS handshake.